### ISSUE TITLE
Error response from daemon: No such container: mongodb

### ISSUE THREAD

**Issue by maxlibin** [zodern/meteor-up#587]
```` -----------------------------------STDERR-----------------------------------
	Error response from daemon: No such container: mongodb
	Error response from daemon: No such container: mongodb
	Error response from daemon: No such container: mongodb
	docker: Error response from daemon: driver failed programming external connectivity on endpoint mongodb (6705f668ba90e3be4512e5236ce9ae73eb2c7f23dd3ec84ac4a3730fcf0b6ae4): Error starting userland proxy: listen tcp 127.0.0.1:27017: bind: address already in use.
	-----------------------------------STDOUT-----------------------------------
	3.4.1: Pulling from library/mongo
	Digest: sha256:aff0c497cff4f116583b99b21775a8844a17bcf5c69f7f3f6028013bf0d6c00c
	Status: Image is up to date for mongo:3.4.1
	Running mongo:3.4.1
	3af6c57cd48ee002accd04d3d484ea51bd78814579794949ce2e3eec1c7fed03
	----------------------------------------------------------------------------
````

any idea whats the problem here, what should i do to fix this?

**Comment by zodern** [comment]
The problem is `Error starting userland proxy: listen tcp 127.0.0.1:27017: bind: address already in use.` There already is something using port 27017.

**Comment by mbabauer** [comment]
I am having the same issue.  I did an both `sudo ss -lptn 'sport = :27017'` and `sudo lsof -n -i :27017` and both came back that nothing was using that port.  @maxlibin, did you ever find the cause of the error?

EDIT: Ok, I got the Mongo step to work.  I had to reboot my server, but I am still having some issues w/ getting the deploy to work, and it looks like all the proxy containers are missing when I do `docker ps`.  Will have to dig further.
